README.md

```diff
@@ -47,9 +91,9 @@ Let's say we have a table "post" with a field "author_id". author_id contains a
 ```javascript
 Arrow.Model.extend('article', {
 	fields: {
-		title: { type: String, model: 'post' },
-		content: { type: String, model: 'post' },
-		author_id: { type: Number, model: 'post' },
+		title: { type: String, model: 'post', name: 'title' },
```

Comment:
why do I have "name" here? Also the order of parameters changed - model is last in the author_first_name, author_last_name......easier to read if they're the same order.